dymos/phase/phase.py

```diff
@@ -1184,7 +1184,7 @@ def add_boundary_constraint(self, name, loc, constraint_name=None, units=None,
         bc_dict[name]['indices'] = indices
         bc_dict[name]['lower'] = lower
         bc_dict[name]['upper'] = upper
-        bc_dict[name]['equals'] = equals
+        bc_dict[name]['equals'] = list(equals) if type(equals) is np.ndarray else equals
         bc_dict[name]['scaler'] = scaler
         bc_dict[name]['adder'] = adder
         bc_dict[name]['ref0'] = ref0
```

Comment:
Also needs a test for adding boundary constraints while specifying each of 'lower', 'upper', and 'equals'.

Reply:
fixed